### PR TITLE
[DO-1886] Add not working converters to report

### PR DESCRIPTION
**Pull request by Iva-khar**

### CODE REVIEW

File: data_ocean/emails.py [hunk 1]
```diff
@@ -2,17 +2,23 @@
...
 
 from data_converter.email_utils import send_template_mail
-from data_ocean.models import Report
+from data_ocean.models import Register, Report
 from users.models import DataOceanUser
 
 
 def send_reports_mail():
+    reports = Report.collect_last_day_reports()
+    not_working = []
```
Comment (criticus) [inline data_ocean/emails.py:11]: please use more explicit name for the variable

File: data_ocean/models.py [hunk 2]
```diff
@@ -92,6 +92,7 @@ class Register(DataOceanModel):
     total_records = models.PositiveIntegerField(_('total records'), default=1, blank=True)
     status = models.CharField(_('status'), max_length=15, choices=STATUSES, default=RELEVANT,
                               blank=True)
+    in_daily_report = models.CharField(max_length=20, null=True)
```
Comment (criticus) [inline data_ocean/models.py:95]: consider renaming to more informative

File: data_ocean/emails.py (hunk 1 above)
Comment (Iva-khar) [reply data_ocean/emails.py]: `not_working ` renamed to `not_updated_registers`

File: data_ocean/models.py (hunk 2 above)
Comment (Iva-khar) [reply data_ocean/models.py]: `in_daily_report` renamed to `name_in_daily_report`

File: data_ocean/models.py [hunk 3]
```diff
@@ -92,6 +92,7 @@ class Register(DataOceanModel):
     total_records = models.PositiveIntegerField(_('total records'), default=1, blank=True)
     status = models.CharField(_('status'), max_length=15, choices=STATUSES, default=RELEVANT,
                               blank=True)
+    name_in_daily_report = models.CharField(max_length=20, null=True)
```
Comment (roman-tiukh) [inline data_ocean/models.py:95]: for char fields better user `blank=True default=''`

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: data_ocean/models.py (hunk 3 above)
Comment (Iva-khar) [reply data_ocean/models.py]: Done

File: data_ocean/emails.py [hunk 4]
```diff
@@ -2,17 +2,23 @@
...
+from data_ocean.models import Register, Report
 from users.models import DataOceanUser
 
 
 def send_reports_mail():
+    reports = Report.collect_last_day_reports()
+    not_updated_registers = []
+    for register in list(Register.objects.exclude(name_in_daily_report='').values('name_in_daily_report')):
+        if all(register['name_in_daily_report'] != report.registry_name for report in reports):
+            not_updated_registers.append(register['name_in_daily_report'])
```
Comment (roman-tiukh) [inline data_ocean/emails.py:14]: maybe?
```suggestion
    not_updated_registers = Register.objects.exclude(
        name_in_daily_report__in=[report.registry_name for report in reports]
    ).exclude(name_in_daily_report='')
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: data_ocean/emails.py (hunk 4 above)
Comment (Iva-khar) [reply data_ocean/emails.py]: changed

File: data_ocean/emails.py [hunk 5]
```diff
@@ -2,17 +2,22 @@
...
 
 
 def send_reports_mail():
+    reports = Report.collect_last_day_reports()
+    not_updated_registers = Register.objects.exclude(
+        name_in_daily_report__in=[report.registry_name for report in reports]
+    ).exclude(name_in_daily_report='')
     send_template_mail(
-        to=settings.REPORT_EMAILS,
+        to=['ivakhar@gmail.com'],
```
[the remark this answers is not in the document]
Comment (Iva-khar) [reply data_ocean/emails.py:15]: )

[the remark this answers is not in the document]
Comment (Iva-khar) [reply data_ocean/emails.py]: fixed

**Review by roman-tiukh — APPROVED** [review]

**Review by PyDev777 — APPROVED** [review]

**Pull request merged**